dev/ansible/04_upload_files.yml

```diff
@@ -58,19 +56,17 @@
   tags: example-upload
 
 - name: Verify that file exist on target `/one/some/prefix/iris-file.csv`
-  shell: mc cat minio-container/one/some/prefix/iris-file.csv
-  failed_when: csv_copy.rc == 1
+  shell: mc ls minio-container/one/some/prefix/iris-file.csv
   tags: example-upload
 
-- name: Upload `json` examples to target `/one/prefix/json/`
+- name: Upload data to target `/one/data/`
   shell: mc cp --recursive ../../example/resources/data/ minio-container/one/data/
```

Comment:
```suggestion
  shell: mc cp --recursive ../../example/resources/data/minio-container/one/data/
```

Reply:
@pdmthorsrud I think it was correct. 
`mc cp --recursive <source> <target>`[doc](https://docs.min.io/docs/minio-client-complete-guide.html#cp)

Reply:
... oops. I'm a dumbdumb :D Sorry!